### PR TITLE
docs: add vega-lite-linter package and vizlinter editor to ecosystem page

### PR DESCRIPTION
**Pull request by qingchentju**
add vega-lite-linter package and vizlinter editor

### CODE REVIEW

File: site/ecosystem.md [hunk 1]
```diff
@@ -25,6 +25,7 @@ We mark featured plugins and tools with a <span class="octicon octicon-star"></s
 - [Codimd](https://github.com/hackmdio/codimd), realtime collaborative markdown notes editor with support of various diagram syntaxes including [Vega-Lite](https://hackmd.io/c/codimd-documentation/%2F%40codimd%2Fextra-supported-syntax#Vega-Lite).
 - [Ivy](http://ivy-vis.netlify.app/), an Integrated Visualization Editing environment that wraps Vega-Lite (among other declarative visualization grammars) as templates to facilitate reuse, exploration, and opportunistic creation. Includes an in-app reproduction of [Polestar](https://github.com/vega/polestar).
 - [Deneb](https://deneb-viz.github.io), a Power BI custom visual with an editor for Vega-Lite or Vega specifications.
+- [VizLinter](https://vizlinter.idvxlab.com/), an online Vega-Lite editor with linting and fixing functions based on vega-lite-linter for Vega-Lite developers to create and validate visualizations.
```
Comment (domoritz) [inline site/ecosystem.md:28]:
```suggestion
- [VizLinter](https://vizlinter.idvxlab.com/), an online Vega-Lite editor with linting and fixing functions based on vega-lite-linter.
```

File: site/ecosystem.md [hunk 2]
```diff
@@ -55,6 +56,7 @@ We mark featured plugins and tools with a <span class="octicon octicon-star"></s
 - [Vega.rb](https://github.com/ankane/vega) brings Vega and Vega-Lite to Ruby.
 - [Jekyll Diagrams](https://github.com/zhustec/jekyll-diagrams) A Jekyll plugin with support for Vega & Vega-Lite and others diagramming libraries.
 - [Liquid Diagrams](https://github.com/zhustec/liquid-diagrams) A Liquid plugin with support for Vega & Vega-Lite and others diagramming libraries.
+- [Vega-Lite-Linter](https://github.com/idvxlab/vega-lite-linter) is a python package to help users detect flaws and rectify already-built but defective visualizations.
```
Comment (domoritz) [inline site/ecosystem.md:59]:
```suggestion
- [Vega-Lite-Linter](https://github.com/idvxlab/vega-lite-linter) is a python package to help users detect and fix encoding issues. 
```

**Review by domoritz — CHANGES_REQUESTED** [review]

**Comment by qingchentju** [comment]
Hi, Dominik! Thank you for the suggested changes. I wonder whether our new package can be marked as featured plugins or tools. 
Thanks again.

Best,
Qing

2021-07-26 22:58:16"Dominik Moritz" ***@***.***>写道：

@domoritz requested changes on this pull request.

In site/ecosystem.md:

> @@ -25,6 +25,7 @@ We mark featured plugins and tools with a <span class="octicon octicon-star"></s

 - [Codimd](https://github.com/hackmdio/codimd), realtime collaborative markdown notes editor with support of various diagram syntaxes including [Vega-Lite](https://hackmd.io/c/codimd-documentation/%2F%40codimd%2Fextra-supported-syntax#Vega-Lite).

 - [Ivy](http://ivy-vis.netlify.app/), an Integrated Visualization Editing environment that wraps Vega-Lite (among other declarative visualization grammars) as templates to facilitate reuse, exploration, and opportunistic creation. Includes an in-app reproduction of [Polestar](https://github.com/vega/polestar).

 - [Deneb](https://deneb-viz.github.io), a Power BI custom visual with an editor for Vega-Lite or Vega specifications.

+- [VizLinter](https://vizlinter.idvxlab.com/), an online Vega-Lite editor with linting and fixing functions based on vega-lite-linter for Vega-Lite developers to create and validate visualizations.


⬇️ Suggested change
-- [VizLinter](https://vizlinter.idvxlab.com/), an online Vega-Lite editor with linting and fixing functions based on vega-lite-linter for Vega-Lite developers to create and validate visualizations.

+- [VizLinter](https://vizlinter.idvxlab.com/), an online Vega-Lite editor with linting and fixing functions based on vega-lite-linter.



In site/ecosystem.md:

> @@ -55,6 +56,7 @@ We mark featured plugins and tools with a <span class="octicon octicon-star"></s

 - [Vega.rb](https://github.com/ankane/vega) brings Vega and Vega-Lite to Ruby.

 - [Jekyll Diagrams](https://github.com/zhustec/jekyll-diagrams) A Jekyll plugin with support for Vega & Vega-Lite and others diagramming libraries.

 - [Liquid Diagrams](https://github.com/zhustec/liquid-diagrams) A Liquid plugin with support for Vega & Vega-Lite and others diagramming libraries.

+- [Vega-Lite-Linter](https://github.com/idvxlab/vega-lite-linter) is a python package to help users detect flaws and rectify already-built but defective visualizations. 


⬇️ Suggested change
-- [Vega-Lite-Linter](https://github.com/idvxlab/vega-lite-linter) is a python package to help users detect flaws and rectify already-built but defective visualizations. 

+- [Vega-Lite-Linter](https://github.com/idvxlab/vega-lite-linter) is a python package to help users detect and fix encoding issues.

**Comment by domoritz** [comment]
We mark things as featured once they are a bit more mature.

**Pull request merged**